packages/babel-preset-gatsby/src/index.js

```diff
@@ -2,13 +2,13 @@ const path = require(`path`)
 
 const resolve = m => require.resolve(m)
 
-const loadCachedConfig = () => {
+const loadCachedConfig = cachePath => {
   let pluginBabelConfig = {}
   if (process.env.NODE_ENV !== `test`) {
     try {
       pluginBabelConfig = require(path.join(
-        process.cwd(),
-        `./.cache/babelState.json`
+        cachePath || path.join(process.cwd(), `./.cache`),
```

Comment:
why `./.cache` on other places with `path.join` you do only a `.cache`

Reply:
it's the same really, I usually kept the original version. But after the resolution it will be `===`

Reply:
also with the `path.join` and `public`